lib/core/rdp.js

```diff
@@ -42,7 +42,18 @@ Rdp.registerGlobalActor = function(client, options) {
     return;
   }
 
-  client.listTabs(response => {
+  let listTabs = {
+    to: "root",
+    type: "listTabs"
+  };
+
+  // NOTE: the following option asks firebug.sdk to mark custom actors registering RDP packets
+  // as rdpInspectorInternals (which helps to filter out them from the packet list)
+  if (options.rdpInspectorInternals) {
```

Comment:
Yes, ```customAttributes``` sounds better.

Also, we could try to use "getTab" packets instead of "listTabs" when possible and keep backward compatibility (not sure when "getTab" has bee introduced). Can be a follow-up.

Honza